draft-ietf-acme-acme.md

```diff
@@ -2512,7 +2512,7 @@ Initial Contents
 
 | Label      | Identifier Type | Reference |
 |:-----------|:----------------|:----------|
-| http-01    | dns             | RFC XXXX  |
+| http-07    | dns             | RFC XXXX  |
```

Comment:
I can agree with using `http-02` for this.

We should probably keep `http-01` in here as well.